org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DebugAdapter.java

```diff
@@ -312,9 +314,27 @@ public DebugAdapter(Consumer<Events.DebugEvent> consumer, IProviderContext conte
     }
 
     private Responses.ResponseBody setBreakpoints(Requests.SetBreakpointArguments arguments) {
+        String clientPath = arguments.source.path;
+        if (AdapterUtils.isWindows() && clientPath.length() > 2) {
+            // VSCode may send drive letters with inconsistent casing which will mess up the key
+            // in the BreakpointManager. See https://github.com/Microsoft/vscode/issues/6268
+            // Normalize the drive letter casing. Note that drive letters
+            // are not localized so invariant is safe here.
+            String drive = clientPath.substring(0, 2);
```

Comment:
We use URIs internally. So the drive letters of different cases won't result in duplicated breakpoints. Do we still need to keep normalizing the drive letter?